### PR TITLE
Prep publish package

### PR DESCRIPTION
**Pull request by sverhoeven**
Follow up on #5 to publish `@classmodel/class` package to jsr.io

<details>
<summary>JSR reports slow types</summary>

```shell
npx jsr publish --dry-run 
Checking for slow types in the public API...
error[missing-explicit-type]: missing explicit type in the public API
 --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:7:7
  | 
7 | const initialState = z
  |       ^^^^^^^^^^^^ this symbol is missing an explicit type
  | 
  = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

error[missing-explicit-type]: missing explicit type in the public API
  --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:29:7
   | 
29 | const timeControl = z
   |       ^^^^^^^^^^^ this symbol is missing an explicit type
   | 
   = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

error[missing-explicit-type]: missing explicit type in the public API
  --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:40:7
   | 
40 | const mixedLayer = z
   |       ^^^^^^^^^^ this symbol is missing an explicit type
   | 
   = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

error[missing-explicit-type]: missing explicit type in the public API
  --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:78:7
   | 
78 | const radiation = z
   |       ^^^^^^^^^ this symbol is missing an explicit type
   | 
   = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

error[missing-explicit-type]: missing explicit type in the public API
  --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:87:14
   | 
87 | export const classConfig = z.object({
   |              ^^^^^^^^^^^ this symbol is missing an explicit type
   | 
   = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

error[missing-explicit-type]: missing explicit type in the public API
  --> /home/stefanv/git/classmodel/class-web/packages/class/src/config.ts:95:14
   | 
95 | export const classDefaultConfigSchema = zodToJsonSchema(
   |              ^^^^^^^^^^^^^^^^^^^^^^^^ this symbol is missing an explicit type
   | 
   = hint: add an explicit type annotation to the symbol

  info: all symbols in the public API must have an explicit type
  docs: https://jsr.io/go/slow-type-missing-explicit-type

This package contains errors for slow types. Fixing these errors will:

  1. Significantly improve your package users' type checking performance.
  2. Improve the automatic documentation generation.
  3. Enable automatic .d.ts generation for Node.js.

Don't want to bother? You can choose to skip this step by
providing the --allow-slow-types flag.

error: Found 6 problems
Child process exited with: 1
```

We could generate type declarations (src/config.d.ts) manually with
```
./node_modules/.bin/tsc  --emitDeclarationOnly --declaration --target esnext --moduleresolution nodenext --module nodenext  src/config.ts 
```
but this gives a chicken (type) and egg (zod) problem.

``

</details>

### CODE REVIEW

File: packages/class/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+../../COPYING
```
Comment (Peter9192) [inline packages/class/LICENSE:1]: Right, so GNU recommends `COPYING` but JSR requires `LICENSE`... 😒

File: packages/class/jsr.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@classmodel/class",
+  "version": "0.0.0",
```
Comment (Peter9192) [inline packages/class/jsr.json:3]:
```suggestion
  "version": "0.0.1",
```

File: packages/class/jsr.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "exports": {
```
Comment (Peter9192) [inline packages/class/jsr.json:4]: This duplicates part of the `package.json`. Can the package exports be omitted from `package.json` if we commit to JSR?

**Review by Peter9192 — APPROVED** [review]
Thanks, nice work!

Testing with `npx jsr publish --dry-run` (without `--allow-slow-types`) highlights some issues that we might still be able to fix quite easily, I think. Perhaps that's already done in #10?

File: packages/class/LICENSE (hunk 1 above)
Comment (sverhoeven) [reply packages/class/LICENSE:1]: ¯\_(ツ)_/¯

File: packages/class/jsr.json (hunk 3 above)
Comment (sverhoeven) [reply packages/class/jsr.json:4]: Nope, thats how you can import the package in the app.

**Pull request merged**